iOS Example Frame/iOS Example FrameUITests/FrameUITests.swift

```diff
@@ -4,7 +4,7 @@
 //
 //  Created by Alex Ioja-Yang on 22/08/2022.
 //  Copyright © 2022 Checkout. All rights reserved.
-//
+// 
```

Comment:
this can be put out of this diff I guess

Reply:
GitHub was unable to load commits today after it had the issues, so had to push something. Will undo 👍